src/Compiler/Compiler.cpp

```diff
@@ -2318,6 +2336,77 @@ bool Compiler::CreateDesign(const std::string& name, const std::string& type) {
   return true;
 }
 
+void Compiler::AddHeaderToLog(const std::filesystem::path& logPath) {
```

Comment:
I think all print stuff should be moved outside the compiler library to foedagcore. Same place as Logger class

Reply:
Moved everything to LogUtils